src/org/jitsi/impl/neomedia/rtp/FrameMarkingHeaderExtension.java

```diff
@@ -105,4 +119,26 @@ public static boolean isEndOfFrame(ByteArrayBuffer baf) {
         byte b = baf.getBuffer()[baf.getOffset() + 1];
         return (b & E_BIT) != 0;
     }
+
+    public static byte getSpatialID(ByteArrayBuffer baf, String encoding) {
+        // Only present on scalable version
+        if (baf == null || baf.getLength() < 4)
+        {
+            return 0;
+        }
+        /*
+         * THIS IS STILL NOT YET CORRECTLY DEFINED ON FRAMEMARKING DRAFT!
+         */
+        return 0;
+    }
+
+    public static byte getTemporalID(ByteArrayBuffer baf) {
+        if (baf == null || baf.getLength() < 2)
```

Comment:
According to the draft (avtext, -4) the TID is also only present in the scalable version. Should we return -1 as an indication that the field is not present if length<4?